source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -345,6 +345,27 @@ def tag_journeys(resp):
     tag_ecologic(resp)
 
 
+def update_durations(pb_resp):
+    """
+    update journey.duration and journey.durations.total
+    """
+    if not pb_resp.journeys:
+        return
+    for j in pb_resp.journeys:
+        j.duration = sum(s.duration for s in j.sections)
+        j.durations.total = j.duration
```

Comment:
Shouldn't we also recompute other things in `j.durations` like `walking/car/bike/etc` ?

Reply:
Good point. As i never found any error on duration per mode i didn't do that.
:+1: